go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/GoogleCloudPlatform/alertmanager
+module github.com/prometheus/alertmanager
```

Comment:
This seems like it will create some confusion about whether references are to our fork or the upstream module.

Reply:
To be clear, all imports in this repository point to `github.com/prometheus/alertmanager`. As such, when I updated `config.go` here, I didn't get the changes in `main.go` because `main.go` actually points to upstream. I want `github.com/prometheus/alertmanager` to point to this repository, so this is what the module name has to be.

This is how we do it on Prometheus as well: https://github.com/GoogleCloudPlatform/prometheus/blob/release-2.45.3-gmp/go.mod

In general, we can't rename the module name because:
1. You'll need to do it for vendored dependencies too.
2. Rebasing on top of upstream would be a PITA because every cherry-picks will fail.

Reply:
Fair enough, makes sense.